src/main/resources/default/assets/scripts/vfs-modals.js

```diff
@@ -194,6 +194,11 @@ function selectVFSFile(config) {
                                         resolve(response.file);
                                     }
                                 });
+                                this.on('maxfilesexceeded', function (file) {
+                                    // replace the file that was uploaded
+                                    this.removeAllFiles();
```

Comment:
Was, wenn maxFiles > 1? Kann das an der Stelle passieren? Soll dann wirklich alles gelöscht werden?

Reply:
In line 127 we configure Dropzone with `maxFiles: 1`, so I don't think this will happen. I also added a comment there referencing this event handler; so in case anyone wants to change this, he should have a look at this line again